debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -114,6 +118,58 @@ public void shouldReceiveChangesForInsertsWithDifferentDataTypes() throws Except
         assertInsert(INSERT_CUSTOM_TYPES_STMT, schemasAndValuesForCustomTypes());
     }
 
+    @Test(timeout=30000)
```

Comment:
Why those timeouts?

Reply:
So, when you initialise the postgis extension it populates the `spatial_ref_sys` system table with 5000-6000 rows (changes each release). I need to do a bit of mucking about to ignore them, and the 30s timeout breaks out of the `while(true)` loop if something doesn't work out.